lib/p2p/Pool.ts

```diff
@@ -123,7 +117,7 @@ class Pool extends EventEmitter {
     const peer = this.peers.get(socketAddress);
     if (peer) {
       peer.destroy();
-      this.logger.warn(`Disconnected from ${peer.pubKey} ${address} ${port}`);
+      this.logger.info(`Disconnected from ${peer.pubKey} ${address} : ${port}`);
```

Comment:
No space after or before the `:` to be consistent with how we display this elsewhere.